lib/rubocop/cop/layout/else_alignment.rb

```diff
@@ -7,6 +7,29 @@ module Layout
       # be aligned with an if/unless/while/until/begin/def keyword, but there
       # are special cases when they should follow the same rules as the
       # alignment of end.
+      #
+      # @example
+      #   # bad
+      #     # case 1
```

Comment:
I think no need to add additional nesting here, just split into separate `# bad` sections. 🙂